### PR TITLE
Paragraph variants 2021

### PR DESCRIPTION
**Pull request by katrialesser**
# READY FOR REVIEW

# Summary
-  Providing paragraph variant options when editing a card paragraph
- It allows different card orientations, border styles, and image styles.

# Review By (Date)
- 4/20/21

# Criticality
- How critical is this PR on a 1-10 scale? Also see [Severity Assessment](https://stanfordits.atlassian.net/browse/D8CORE-1705).
- E.g., it affects one site, or every site and product?

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out the site https://saauserguide-test.sites.stanford.edu/ and other sites on the paragraph-variants-2021 branch
2. Navigate to pages with icon cards, make sure they are completely fine, even though they haven't been edited to use paragraph variants yet (see Jen's ticket: see adapt-1900
3. We have a lot of testing tickets in jira we are going through for this big pr:
https://stanfordits.atlassian.net/browse/ADAPT-1900
4. Test to make sure we haven't affected banners: see https://stanfordits.atlassian.net/browse/ADAPT-1900

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: templates/components/pattern-card.html.twig [hunk 1]
```diff
@@ -7,6 +7,8 @@
  */
 #}
 {# until interface options are available, all cards will be icon cards on SAA sites #}
-{% set attributes = attributes.addClass('su-card--icon pt-0 text-left') %}
+{#{% set attributes = attributes.addClass('su-card--icon pt-0 text-left') %}#}
```
Comment (katrialesser) [inline templates/components/pattern-card.html.twig:10]: we could remove this comment now or later..

**Review by katrialesser — COMMENTED** [review]
I think it is looking good.

**Review by rebeccahongsf — APPROVED** [review]
Tested out locally and on our SAA test sites. Everything looks great and works as intended. 🚢

**Review by jenbreese — APPROVED** [review]
I think it looks good.

**Pull request merged**